blog/2023-01-18-introducing-netdata-paid-plans/index.md

```diff
@@ -0,0 +1,43 @@
+---
+slug: introducing-netdata-paid-subscriptions
+title: Introducing Netdata Paid Subscriptions
+description: All Netdata functionality is and will be available for free in the Free Community Plan. Paid tiers include features targeted for businesses of different sizes and with different monitoring requirements.
+authors: team
+tags: [pricing, free-plan, paid-subscriptions]
+keywords: [pricing, free plan, paid subscriptions, monitoring, troubleshoot]
+image: ./img/subscriptions.png
+
+---
+
+Read more about Netdata introducing paid subscriptions. All Netdata functionality is and will be available for free in the Free Community Plan. Paid tiers include features targeted for businesses of different sizes and with different monitoring requirements.
```

Comment:
It seems a bit odd to say "**_All_** Netdata functionality is and will be free in community plan"

shouldn't there be some qualifier? All the Netdata functionality you currently use or something along similar lines?